### ISSUE TITLE
TINET Repo Mirror inquiry

### ISSUE THREAD

**Issue by alessiodam** [KingDubDub/Jetpack-Joyride-CE#5]
I would like to ask if it was possible to mirror this game over to TINET.  
This will make the users have the possibility to download your game from their calculator so they can be even more Super-Cool Tech Wizz 🔥🔥🔥

**Comment by KingDubDub** [comment]
I've just uploaded it and while it did work it seems that buckets and account info haven't been implemented on the calc side. I'll also look into updating this to use more modern functions when I have time, there's a few fixes and small changes I haven't pushed and was messing with a while back.

**Comment by alessiodam** [comment]
Thanks, I would like to point out that you don't need to upload to buckets but that the mirror will point to the latest release URL and download the .8xp file, when creating new releases, please put a JOYRDE.8xp so the calculator could download it directly from github.

**Comment by alessiodam** [comment]
I also look forward to more updates of this game! Hope there will be :)

**Comment by alessiodam** [comment]
Also, if you are able to, try to make 1 file for the game otherwise it won't work ;)

**Comment by KingDubDub** [comment]
I'm afraid that is entirely impossible, the max file size for the data is exceeded a minimum of 4 times over when everything is compressed. I'll be looking into using zx0 compression though, it seems to be halving the time to decompress while making it fairly smaller so it's worth a shot from at least a performance standpoint to see what I can do.

**Comment by alessiodam** [comment]
Ok, I'll wait for updates then np ;)

**Comment by KingDubDub** [comment]
Closing since TINET died years ago, such a shame.

**Comment by alessiodam** [comment]
Wdym years ago 😭 (+ it's not a shame I'm working on [TIgol](https://tigol.net) :/)

**Comment by KingDubDub** [comment]
Oh neat, if this is a souped-up version of TINET then poke me when I can upload programs there. Good to see you're still around.

**Comment by alessiodam** [comment]
Yeah TINET was really bad. It was my first project. Now I'm back with much much more experience xD
